static/src/stylesheets/module/email/_form.scss

```diff
@@ -3,10 +3,12 @@
 }
 
 .email-sub__iframe {
-    // pixel calculations are manual design 
-    // adjustments, off grid
-    width: calc(100% + 8px);
-    margin-left: -4px;
+    @if ($old-ie) {
+        width: 100%;
+    } @else {
+        width: calc(100% + 8px);
```

Comment:
I shouldn't have removed that comment :)

> // pixel calculations are manual design 
> // adjustments, off grid

@superfrank was making adjustments based on the curve of the earth or something.